sematic/ui/src/components/ScrollingLogView.tsx

```diff
@@ -74,6 +69,21 @@ export default function ScrollingLogView(props: {
     return "Rendering...";
   }, [isAccumulating, isAccumulatorLoading, accumulatedLines, hasMore]);
 
+  const pullDownTriggerSection = useMemo(() => {
+    if (!pullDownTriggerEnabled || isAccumulating || !hasMore) {
+      return <></>;
+    }
+    return (<>
+      <span style={{marginLeft: "2em", fontSize: '14px'}}>Keep scrolling down to get more logs</span>
```

Comment:
Can we make this look nicer? Maybe use a:
```suggestion
      <Alert severity="info" icon={<ArrowCircleDownIcon fontSize="inherit" />}>Keep scrolling down to get more logs</Alert>
```
https://mui.com/material-ui/material-icons/?query=down&selected=ArrowCircleDown

Reply:
Yes, that looks great. I adopted that. 👍